heltour/tournament/templates/tournament/player_profile.html

```diff
@@ -193,6 +194,35 @@ <h3>Career Statistics</h3>
 			</div>
 		</div>
 		{% endif %}
+        {% if trophies %}
+		<div class="well">
+			<div class="well-head">
+				<h3>Trophy Case</h3>
```

Comment:
If this is only showing trophies from one league, the UI should make that clear. For example, change `Trophy Case` to `{{ league.name }} Trophies`.